src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public Figure create() {
+        Circle circle = new Circle();
+        circle.radius = getRandom().nextInt(getMaxSize());
+        circle.setColor(getColorSupplier().getRandomColor());
+        return circle;
+    }
```

Comment:
This method should not be prescribed in classes.  You have to create a random shape in FigureSupplier getRandomFigure